sdk/requirements.txt

```diff
@@ -4,9 +4,9 @@ fire
 google-auth>=1.30.0,<2.0dev
 google-cloud-storage>=1.37.0
 mlflow==1.14.1
-numpy==1.21.6
-pandas==1.2.2
-protobuf==3.20.1
-python-dateutil
-requests==2.25.1
-urllib3 >= 1.25.3
+numpy
+pandas
+protobuf>=3.0.0,<4.0.0
+python_dateutil>=2.5.3
+requests
+urllib3>=1.25.3
```

Comment:
Aligned these changes with those of merlin: https://github.com/gojek/merlin/blob/main/python/sdk/setup.py
Note that the `numpy` and `pandas` versions are not pinned either.